### ISSUE TITLE
flash attention

### ISSUE THREAD

**Issue by Charlie839242** [Pointcept/PointTransformerV3#12]
Hello and thank you for your amazing work!

I am wondering that will enabling flashattention reduce the model performance or not? 

Thanks for your reply!

**Comment by Gofinge** [comment]
Hi, typically not. During our research, we initially did not support flash attention and enable RPE in our model. A significant performance improvement occurs when we enlarge the patch size from 16 to 128. Then we ablated disabling RPE and found that it has an ignorable influence on the final performance (except S3DIS). Then we support FlashAttention (which needs to disable RPE) and further enlarge the patch size to 1024 or higher, which is impossible when disabling FlashAttention, and leading to a better performance.

So in conclusion, enabling flash attention will make a larger patch size available which is typically good for the model performance (except a few cases need RPE, yet the gap, in my opinion, is not so important compared with the efficiency gain)

**Comment by Charlie839242** [comment]
Thanks for giving so many details, It helps me a lot!

**Comment by Merealtea** [comment]
May I ask a question? How to disable the flashattention? My server doesn`t have a new version GPU

**Comment by Merealtea** [comment]
> May I ask a question? How to disable the flashattention? My server doesn`t have a new version GPU

Never Mind, I found it in the configure file
